src/bp/common/typings.ts

```diff
@@ -17,6 +17,18 @@ export interface UniqueUser {
   strategy: string
 }
 
+export interface CreatedUser {
+  email: string
+  tempPassword: string
+}
+
+export interface WorkspaceUser {
```

Comment:
Should we remove the interface WorkspaceUser in workspace_users.ts and use this one instead?